thorlcr/msort/tsorts1.cpp

```diff
@@ -71,7 +71,7 @@ class CMergeReadStream : public CSimpleInterface, public IRowStream
         {
             Owned<ISecureSocket> ssock = secureContextClient->createSecureSocket(socket.getClear());
             int tlsTraceLevel = SSLogMin;
-            if (sortTraceLevel >= MPVerboseMsgThreshold)
+            if (sortTraceLevel > ExtraneousMsgThreshold)
```

Comment:
shouldn't this be >= ? i.e. if I set sortTraceLevel to exactly ExtraneousMsgThreshold, wouldn't I expect to see this tracing?